### PR TITLE
Feature/covid_status_by_state

### PR DESCRIPTION
**Pull request by robinhester**
updating issues with covid api, needed to add in correct import statements that were missed. Issues fixed.

### CODE REVIEW

File: project/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
-uvicorn==0.11.8
\ No newline at end of file
+pandas==1.1.3
+plotly==4.12.0
+uvicorn==0.11.8
+sodapy==2.1.0
+scikit-learn==0.23.2
+requests==2.24.0
+python-dotenv == 0.15.0
+
```
Comment (AmyBeisel) [inline project/requirements.txt:9]: Thanks for adding the required libraries!

File: project/app/api/predict.py [hunk 2]
```diff
@@ -53,3 +61,25 @@ async def predict(item: Item):
...
+    df = pd.DataFrame.from_records(results)
+    state_requested = pd.DataFrame([state])
+    state = state_requested.iloc[0][0]
+
+    last_week = str(datetime.date.today() - timedelta(days = 7))
+
+    # filter df for above info and get the total
+    new= df[(df['state'] == state) & (df['submission_date'] > last_week)]
+    new_cases= new['new_case'].astype('float').sum()
+    return new_cases
```
Comment (AmyBeisel) [inline project/app/api/predict.py:85]: Looks good. Will follow Pep8 next time.

**Review by AmyBeisel — APPROVED** [review]
Covid API up and running. Getting ready to deploy to AWS which is why we added all the requirements.  Deleted the Dockerrun.aws.json file as well to deploy to AWS.

File: project/app/api/predict.py (hunk 2 above)
Comment (richardOlson) [reply project/app/api/predict.py:85]: Looks good for the covid information API.

**Review by richardOlson — APPROVED** [review]
We talked about the Docker file being deleted and Robin will update the PEP 8 violation.(The formating of the empty line below the return statement.)

**Pull request merged**